### PR TITLE
Doc Changes

### PR DESCRIPTION
**Pull request by ntyukaev**

### CODE REVIEW

File: demos/README.md [hunk 1]
```diff
@@ -113,10 +113,10 @@ Please run the following command before the demos build (assuming that the binar
 source <INSTALL_DIR>/deployment_tools/bin/setupvars.sh
 ```
 You can also build demos manually using Inference Engine binaries from the
-[dldt](https://github.com/opencv/dldt/tree/master) repo. In this case please set `InferenceEngine_DIR` to a CMake folder you built the dldt project from, for example `<dldt_repo>/inference-engine/build`.
+[openvino](https://github.com/openvinotoolkit/openvino) repo. In this case please set `InferenceEngine_DIR` to a CMake folder you built the Inference Engine from, for example `<openvino_repo>/inference-engine/build`.
 Please also set the `OpenCV_DIR` variable pointing to the required OpenCV package. The same OpenCV
 version should be used both for the inference engine and demos build.
-Please refer to the Inference Engine [build instructions](https://github.com/opencv/dldt/blob/master/inference-engine/README.md)
+Please refer to the Inference Engine <a href="https://github.com/openvinotoolkit/openvino/blob/master/build-instruction.md">build instructions</a>
```
Comment (IRDonch) [inline demos/README.md:119]: The URL change is fine, but why did you turn it into an HTML link?

File: demos/python_demos/colorization_demo/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ You can use the following models with the demo:
 * `colorization-v2`
 * `colorization-v2-norebal`
 
-For more information about pretrained models, refer to the [model documentation](../../../models/public/index.md).
+For more information about the pre-trained models, refer to the [model documentation](../../../models/intel/index.md).
```
Comment (IRDonch) [inline demos/python_demos/colorization_demo/README.md:9]: These are actually public models, not Intel models.

File: demos/python_demos/human_pose_estimation_3d_demo/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ This demo demonstrates how to run 3D Human Pose Estimation models using OpenVINO
 
 * `human-pose-estimation-3d-0001`.
 
-For more information about the pre-trained models, refer to the [model documentation](../../../models/public/index.md).
+For more information about the pre-trained models, refer to the [model documentation](../../../models/intel/index.md).
```
Comment (IRDonch) [inline demos/python_demos/human_pose_estimation_3d_demo/README.md:7]: Same here. The model is in the `public` directory.

File: demos/README.md (hunk 1 above)
Comment (andrew-zaytsev) [reply demos/README.md:119]: It's just a workaround to get this link working in the generated HTML documentation. This affects all external (not included to the documentation project) .md files.

File: demos/python_demos/colorization_demo/README.md (hunk 2 above)
Comment (andrew-zaytsev) [reply demos/python_demos/colorization_demo/README.md]: fixed

File: demos/python_demos/human_pose_estimation_3d_demo/README.md (hunk 3 above)
Comment (andrew-zaytsev) [reply demos/python_demos/human_pose_estimation_3d_demo/README.md]: fixed

File: demos/README.md (hunk 1 above)
Comment (IRDonch) [reply demos/README.md:119]: OK. I assume you're talking about the doxygen bug fixed by doxygen/doxygen#6913.

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**